graspologic/embed/omni.py

```diff
@@ -2,15 +2,55 @@
 # Licensed under the MIT License.
 
 import warnings
-from typing import Optional
+from typing import List, Optional
 
 import numpy as np
-from scipy.sparse import isspmatrix_csr
+from scipy.sparse import csr_matrix, hstack, isspmatrix_csr, vstack
 
-from ..utils import import_graph, is_fully_connected
+from ..utils import average_matrices, import_graph, is_fully_connected
 from .base import BaseEmbedMulti
 
 
+def _get_omnibus_matrix_sparse(matrices: List[csr_matrix]) -> np.ndarray:
```

Comment:
This is very similar to what I implemented in topologic

Reply:
is the `ndarray` returned from this fn sparse or dense?

Reply:
I think I see the issue - `scipy.vstack` returns a `scipy.sparse.csr_matrix` if you pass it a `csr_matrix`, not an `ndarray`

Reply:
Updated